openssl-sys/src/handwritten/params.rs

```diff
@@ -1,8 +1,14 @@
 use super::super::*;
 use libc::*;
 
+#[cfg(ossl300)]
```

Comment:
I don't think this is required, the entire module has this cfg.

Reply:
Yep, removed - rebase conflict!